pyingest/parsers/author_init.py

```diff
@@ -0,0 +1,17 @@
+import namedentities
+from pyingest.config.utils import u2asc
+
+class AuthorInitial(object):
+
+     def __init__(self):
+         pass
+
+     def get_author_init(self, namestring):
+         try:
+             instring = namedentities.unicode_entities(namestring)
+             outstring = u2asc(instring)[0]
+             if outstring.isalpha():
+                 return outstring
+         except Exception as err:
```

Comment:
If the first part throws an exception, it fails silently. It would be nice to know the actual exception. As a general pattern, we should avoid code that fails silently and just "passes".

Reply:
Agreed -- I added a custom exception called BadAuthorInitialException to make the origin obvious, and passed the error message.